app/views/companies/index.html.erb

```diff
@@ -0,0 +1,20 @@
+<h3>Partner Companies</h3>
+<hr/>
+
+<ul class="company-list">
+  <% @companies.each do |company| %>
+    <li>
+      <%= link_to company.name, company %>
+      <span class='update-company'>
+        <%= link_to 'Edit', edit_company_path(company) %>
+        <%= spacer %>
+        <%= link_to 'Delete', company_path(company), method: :delete %>
+        </span>
+    </li>
+    <% unless company.last_company? %>
+      <hr>
+    <% end %>
+  <% end %>
+</ul>
+<hr>
+<%= link_to "+ New Company", new_company_path, class: "unstyled-link" %>
```

Comment:
I think else where we say "Add resource name", rather than "+ Resource Name". Double-check and then let's be consistent.